lib/src/inline_parser.dart

```diff
@@ -885,16 +899,19 @@ class TagSyntax extends InlineSyntax {
   /// it on strikethrough.
   final bool allowIntraWord;
 
-  /// Create a new [TagSyntax] which matches text on [pattern].
+  final List<DelimiterTag>? tags;
+
+  /// Create a new [DelimiterSyntax] which matches text on [pattern].
```

Comment:
Nit: "Creates" instead of "Create"

https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs

Reply:
Thanks! Fixed